### PR TITLE
Fix paths on Alpine

### PR DESCRIPTION
**Pull request by Kakadu**
Continuation of #9915

### CODE REVIEW

File: packages/conf-qt/conf-qt.5.2.1/opam [hunk 1]
```diff
@@ -6,9 +6,15 @@ bug-reports:  "https://github.com/kakadu/lablqt/issues"
 dev-repo:     "https://github.com/Kakadu/lablqt.git"
 
 build: [
-    ["sh" "-exc" "echo \"Your Qt version is `qmake -query QT_VERSION`\"" ]
-    ["sh" "-exc" "min=\"5.2.1\"; cur=`qmake -query QT_VERSION`; res=`printf \"$min\\n$cur\\n\" | sort | head -n 1`; [ \"$res\" = \"$min\" ]"
-    ]
+    ["sh" "-exc" "echo \"Your Qt version is `qmake -query QT_VERSION`\"" ] 
+    { os != "alpine" && os != "centos" }
+    ["sh" "-exc" "min=\"5.2.1\"; cur=`qmake -query QT_VERSION`; res=`printf \"$min\\n$cur\\n\" | sort | head -n 1`; [ \"$res\" = \"$min\" ]" ] 
+    { os != "alpine" && os != "centos" }
```
Comment (yallop) [inline packages/conf-qt/conf-qt.5.2.1/opam]: The `&&` here and above should be `&`.

**Comment by Kakadu** [comment]
@yallop should I squash everything to a single commit that adjusts path for ALL Linuxes ? Should we merge it as it is?

**Comment by yallop** [comment]
@Kakadu: yes, adjusting the path for all Linuxes is a good idea.  Could you please make that change and squash?

**Comment by Kakadu** [comment]
@yallop done

**Pull request merged**